ARCL/Source/SceneLocationView.swift

```diff
@@ -111,7 +111,7 @@ public class SceneLocationView: ARSCNView, ARSCNViewDelegate {
         showsStatistics = false
 
         if showFeaturePoints {
-            debugOptions = [ARSCNDebugOptions.showFeaturePoints]
+            debugOptions = [SCNDebugOptions.showFeaturePoints]
```

Comment:
Why did you change this from `ARSCNDebugOptions` to `SCNDebugOptions`?

I'm not saying it's wrong, but we're using `ARSCNView` objects (which are more specific than an `SCNView`).